components/Block.vue

```diff
@@ -75,66 +75,56 @@ export default {
     },
     handleDragStart(e) {
       const { clientX, clientY, offsetX, offsetY } = e
-      const { offsetHeight, offsetWidth } = e.target.offsetParent.parentElement
+      const boxNode = e.target.parentNode.parentNode.getBoundingClientRect()
+      const { height, width } = boxNode
 
       this.ghostDraggingStyle = {
-        height: `${offsetHeight}px`,
-        width: `${offsetWidth}px`,
+        height: `${height}px`,
+        width: `${width}px`,
         top: `${clientY}px`,
         left: `${clientX}px`,
         transform: `translate(${-Math.abs(offsetX)}px,${-Math.abs(offsetY)}px)`
       }
 
-      this.dragStartCoords.x = clientX
-      this.dragStartCoords.y = clientY
       this.isDragging = true
     },
     handleDrag(e) {
-      e.preventDefault()
       const { clientX, clientY } = e
+      const boxNode = e.target.parentNode.parentNode.getBoundingClientRect()
+      const { x, y } = boxNode
 
       this.ghostDraggingStyle = {
         ...this.ghostDraggingStyle,
         top: `${clientY}px`,
         left: `${clientX}px`
       }
 
-      const movementX = clientX - this.dragStartCoords.x
-      const movementY = clientY - this.dragStartCoords.y
-
-      if (movementX > 40) {
-        this.dragStartCoords.x = clientX
+      const movementX = clientX - x
+      const movementY = clientY - y
 
+      if (movementX > 70) {
         if (this.blockData.x < 11) {
           this.blockData.x += 1
         }
-      } else if (movementX < -40) {
-        this.dragStartCoords.x = clientX
-
+      } else if (movementX < -70) {
         if (this.blockData.x > 0) {
           this.blockData.x -= 1
         }
       }
 
-      if (movementY > 20) {
-        this.dragStartCoords.y = clientY
-
+      if (movementY > 50) {
         if (this.blockData.y < 2) {
           this.blockData.y += 1
         }
-      } else if (movementY < -20) {
-        this.dragStartCoords.y = clientY
-
+      } else if (movementY < -50) {
         if (this.blockData.y > 0) {
           this.blockData.y -= 1
         }
       }
     },
-    handleDragEnd(e) {
-      e.preventDefault()
-      this.dragStartCoords = { x: 0, y: 0 }
-      this.ghostDraggingStyle = {}
+    handleDragEnd() {
```

Comment:
I'm thinking the conditionals here might be what's giving you the janky stuff on resizing. I want to play with this haha

Reply:
oops meant the (x, y) subtractions/updates

Reply:
pull it down and play with it! i might actually end up using this for the admin side of another project